### DISCUSSION TITLE
I want to create Blog Pages for Each Topic and Link to Home Page's Cards

### DISCUSSION THREAD

**Discussion by satyasootar** [ANSHIKA-26/WordWise#472] [Ideas]
So that when a user click on the respected topic's card . It will redirect it to the blog page.
@ANSHIKA-26

**Comment by ANSHIKA-26** [comment]
explain it a little ?

**Comment by satyasootar** [comment]
I am planning to develop comprehensive blog posts for each of the 14 topics featured on the homepage. These blogs will provide in-depth insights and valuable information on the respective topics. Once completed, each blog will be linked to the corresponding card on the homepage. This will enable users to easily click on a topic card and be redirected to the full blog post, ensuring a seamless and engaging user experience. The goal is to enhance content accessibility and provide detailed resources on each subject directly from the homepage.




https://github.com/user-attachments/assets/05cc55a7-5363-43da-b804-db7c82e41572



@ANSHIKA-26

**Reply by ANSHIKA-26** [reply]
this wont improve the functionality/features of the website in any way :( , just increase the number of files we have

**Comment by ANSHIKA-26** [comment]
honestly , those hardcoded blogs arent something i wish to add to the website , any other idea you wish to work upon  something like a user writes a blog , its stored in backend and is fetched from there to frontend after user clicks on home page cards ?
its better than the blogs being put inside the code ( the way it is right now ) 
i am open to suggestions !
